ios/RNSnackBarView.m

```diff
@@ -175,6 +176,7 @@ - (void)show {
     }
     if (!_pendingOptions) { return; }
 
+    self.backgroundColor = _pendingOptions[@"backgroundColor"] ? [RCTConvert UIColor:_pendingOptions[@"backgroundColor"]] : [UIColor colorWithRed:0.196078F green:0.196078F blue:0.196078F alpha:1.0F];
```

Comment:
Could you please [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) this up a bit and make the default `UIColor` here a constant?